### PR TITLE
docs: updates windows/mac install topics

### PR DESCRIPTION
**Pull request by chri2547**
This PR updates and restructures the install on windows and mac topics.

### CODE REVIEW

File: docs/sources/setup-grafana/installation/mac/index.md [hunk 1]
```diff
@@ -29,44 +31,36 @@ Use [Homebrew](http://brew.sh/) to install the most recent released version of G
    - `/usr/local/Cellar/grafana/[version]` (Homebrew v2)
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/mac/index.md:31]: @DanCech , Are these homebrew versions still valid? Are there others we should list?

Comment (DanCech) [reply docs/sources/setup-grafana/installation/mac/index.md:31]: The current homebrew version is v4, and the install path actually seems to depend on the hardware itself, `/usr/local/` for Intel silicon and `/opt/homebrew/` for Apple silicon.
```suggestion
   - `/usr/local/Cellar/grafana/[version]` (Intel Silicon)
```

File: docs/sources/setup-grafana/installation/mac/index.md [hunk 2]
```diff
@@ -29,44 +31,36 @@ Use [Homebrew](http://brew.sh/) to install the most recent released version of G
...
    - `/opt/homebrew/Cellar/grafana/[version]` (Homebrew v3)
```
Comment (DanCech) [inline docs/sources/setup-grafana/installation/mac/index.md:32]:
```suggestion
   - `/opt/homebrew/Cellar/grafana/[version]` (Apple Silicon)
```

File: docs/sources/setup-grafana/installation/mac/index.md [hunk 3]
```diff
@@ -1,23 +1,25 @@
 ---
 aliases:
   - ../../installation/mac/
-description: Installing Grafana on macOS
-title: Install on macOS
+description: How to install Grafana OSS and Enterprise on macOS
```
Comment (DanCech) [inline docs/sources/setup-grafana/installation/mac/index.md:4]:
```suggestion
description: How to install Grafana OSS or Enterprise on macOS
```

File: docs/sources/setup-grafana/installation/windows/index.md [hunk 4]
```diff
@@ -1,49 +1,63 @@
...
    - The most recent Grafana version is selected by default.
-   - The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
+   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
 1. Select an **Edition**.
-   - **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
-   - **Open Source** - Functionally identical to the enterprise version, but you will need to download the enterprise version if you want enterprise features.
+   - **Enterprise:** This is the recommended version. It is functionally identical to the open source version, but includes features you can unlock with a license, if you so choose.
+   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
 1. Click **Windows**.
+1. To use the Window installer, complete the following steps:
```
Comment (DanCech) [inline docs/sources/setup-grafana/installation/windows/index.md:22]:
```suggestion
1. To use the Windows installer, complete the following steps:
```

File: docs/sources/setup-grafana/installation/mac/index.md [hunk 5]
```diff
@@ -26,47 +28,39 @@ Use [Homebrew](http://brew.sh/) to install the most recent released version of G
...
-   ```bash
-   curl -O https://dl.grafana.com/oss/release/grafana-7.1.5.darwin-amd64.tar.gz
-   ```
-1. Untar the gz file and copy the files to the location of your preference.
+1. Copy and paste the code from the installation page into your command line and run.
+1. Untar the `gz` file and copy the files to the location of your preference.
 1. To start Grafana service, go to the directory and run the command:
+
    ```bash
    ./bin/grafana-server web
```
Comment (DanCech) [inline docs/sources/setup-grafana/installation/mac/index.md:57]:
```suggestion
   ./bin/grafana-server
```

File: docs/sources/setup-grafana/installation/windows/index.md [hunk 6]
```diff
@@ -1,49 +1,63 @@
 ---
 aliases:
   - ../../installation/windows/
-description: Install Grafana on Windows
-title: Install on Windows
+description: How to install Grafana OSS and Enterprise on Windows
```
Comment (DanCech) [inline docs/sources/setup-grafana/installation/windows/index.md:4]:
```suggestion
description: How to install Grafana OSS or Enterprise on Windows
```

**Review by DanCech — APPROVED** [review]

**Comment by zerok** [comment]
Hi :) Should this be backported to 9.5.x and 9.4.x? Right now it's only in main (targetting 10.0.0).

**Comment by DanCech** [comment]
it wouldn't hurt to backport it

**Comment by DanCech** [comment]
no need to backport this to 9.3.x imho

**Pull request merged**